web/themes/custom/my_awesome_theme/templates/views/views-view--recent-articles.html.twig

```diff
@@ -0,0 +1,21 @@
+<div  class="recent-posts">
+  <h4>
+    <span>
+      RECENT POSTS
```

Comment:
Need to use translatable rows. You can use a trans block or |t.
E.G. Recent Posts |t